packages/extension/src/background/transaction/actions.ts

```diff
@@ -183,17 +178,7 @@ export function getLedgerFromMixedGenesis(genesisId: string, genesisHash: string
       }
     }
 
-    // Injected networks may have additional information, multiples, or additional checks
-    // so we will check them separately
-    const injectedNetworks = Settings.getCleansedInjectedNetworks();
-    injectedNetworks.forEach(network => {
-      if (network['genesisHash'] === genesisHash) {
-        // Found genesisHash, make sure the id matches 
-        if (!genesisId || genesisId === ledger.genesisId) { 
-          return network;
-        }
-      }
-    });
+    // We don't currently store the genesisHash of the custom networks
```

Comment:
Should we not still check hash here and add the storage capability on the custom networks page?

Reply:
This is a temporary workaround until we add support for storing hashes on custom networks in a future release.